IdentityCore/src/webview/embeddedWebview/challangeHandlers/mac/MSIDCertAuthHandler.m

```diff
@@ -106,6 +106,7 @@ + (void)promptUserForIdentity:(NSArray *)issuers
     NSMutableDictionary *query =
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->